docs/source/usage/deployment.rst

```diff
@@ -0,0 +1,118 @@
+
+  1. Login to AWS Console using your account credentials. This framework has been tested to work with federated logins where the user assumes a role.
+  2. Navigate to “Services” and under “Management and Governance” select “Systems Manager”
+  3. Once the page loads, under “Application Management, choose “Parameter Store"
+  4. Create a parameter there in the form of:
+
+       “/<Project>/<service>/<stage>/auth_token”
+
+       **Example:** For our Zoom example, this would be something like:
+       
+    .. image:: ../images/Parameter_Store_Example.png
+       :width: 400
+       :alt: /MozDef-EF/zoom/prod/auth_token
+
+  5. In the description field, add “Authentication token used by the Webhook app to post events”.
+  6. For type, select “SecureString”. 
+  7. For value, paste the value from the app’s configuration from your webhook's configured authentication token.
+  8. Add a Tag with key: Project and value: MozDef-EF-<environment>, then create the parameter.
+
+*************************************
+Fill in the Framework Config Template
+*************************************
+
+The following should be done in your local copy of the framework you cloned or forked:
+
+
+  1. Open the downloaded repository in an IDE to edit locally.
+  2. Under config directory, edit “buildspec-prod.yml” file to contain:
+
+    .. code-block:: yaml
+    
+       version: 0.2
+       
+       env:
+         variables:
+           STAGE: prod
+           SERVICE: zoom
+           PROJECT: MozDef-EF
+           API_PATH: zoom
+       
+       phases:
+         install:
+           runtime-versions:
+             python: 3.7
+             nodejs: 10
+           commands:
+             # Install dependencies here
+             - pip3 install --upgrade awscli -q
+             - pip3 install --upgrade pytest -q
+             - pip3 install --upgrade moto -q
+             - pip3 install --upgrade aws-xray-sdk -q
+             - npm install -g --silent --progress=false serverless
+             - npm install --silent --save-dev serverless-pseudo-parameters
+             - npm install --silent --save-dev serverless-prune-plugin
+         pre_build:
+           commands:
+             # Perform pre-build actions here
+             - chmod +x $CODEBUILD_SRC_DIR/config/deploy.sh
+             - $CODEBUILD_SRC_DIR/config/deploy.sh unit-test
+         build:
+           commands:
+             # Invoke the deploy script here
+             - $CODEBUILD_SRC_DIR/config/deploy.sh deploy $STAGE $AWS_REGION
+
+
+  3. The important part here is the filling in of the “env” section on top. These environment variables will be used by the “serverless.yml” file when deployed by the serverless framework.
+     The idea is, for each different source (such as zoom), the service name and API path will be different.
+  4. Save the file.
+  5. Make any other desired changes on the local copy. For the zoom service, there should not be any other changes.
```

Comment:
I wonder, should we generalise and say "For webhook-based event sources (such as Zoom), there should not be any other changes." instead of "For the zoom service, ...".

Just an idea.

Reply:
Haha you wrote that, I just copied and pasted :p

We can definitely modify, this is exactly the feedback I was looking for  :) ty

Reply:
check out what I've added and let me know if that works for you :)

Reply:
Yeah I know I wrote the most of this originally :) This is a way for me to review what I wrote as well. I figured, before it was documented rather less formally on a Gdoc, and a proper documentation deserves a better explanation.

Reply:
Yeah changes look great. Ty!